.eslintrc.js

```diff
@@ -1,5 +1,22 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+/* eslint-env node */
+
+const path = require('path');
+
 module.exports = {
 	extends: 'liferay',
 	plugins: [],
-	rules: {},
+	rules: {
+		'notice/notice': [
+			'error',
+			{
+				templateFile: path.join(__dirname, 'copyright.js'),
+			},
+		],
```

Comment:
New version of eslint-config-liferay requires this be set up explicitly now rather than assuming a default (see https://github.com/liferay/eslint-config-liferay/issues/115 for details but the TL;DR is that we needed to make it that way so that we could use two different licenses in the liferay-portal repo).